frontend/src/components/common/SideBar.tsx

```diff
@@ -62,6 +68,46 @@ export default function SideBar() {
     router.push(route);
   };
 
+  const handleLoggingOut = () => {
```

Comment:
I think we can also clear the cookies when logging out.

Reply:
Resolved